tests/playground.py

```diff
@@ -47,7 +47,13 @@
 wiz.res("h", 42, 13.0, 24.0)
 wiz.res("h&", 42, 13.0, 24.0)
 wiz.res("i", Decimal("42.0e-30"), Decimal("0.1e-31"), r"\m")
-wiz.res("i", Decimal("42.0e-30"), Decimal("0.1e-31"), Decimal("0.05e-31"), r"\m\per\s\squared")
+wiz.res(
+    "i",
+    Decimal("42.0e-30"),
+    Decimal("0.1e-31"),
+    Decimal("0.05e-31"),
+    r"\m\per\s\squared\newton\per\kg",
+)
```

Comment:
Would you mind waiting for #41, then implementing some test cases for this new parsing logic?

Reply:
Will resolve this for now and we'll add tests along the way once we're fully happy with the API, which is still a bit under development.